systemtest/src/test/java/io/strimzi/systemtest/operators/CustomResourceStatusST.java

```diff
@@ -344,15 +327,12 @@ void testKafkaMirrorMaker2Status() {
                 KafkaNodePoolTemplates.controllerPoolPersistentStorage(testStorage.getNamespaceName(), testStorage.getControllerPoolName(), testStorage.getClusterName(), 1).build()
             )
         );
-        resourceManager.createResourceWithWait(KafkaTemplates.kafkaPersistent(testStorage.getClusterName(), 1, 1)
+        resourceManager.createResourceWithWait(KafkaTemplates.kafkaPersistent(testStorage.getNamespaceName(), testStorage.getClusterName(), 1, 1)
             .editMetadata()
                 .withNamespace(Environment.TEST_SUITE_NAMESPACE)
             .endMetadata()
```

Comment:
likewise.

Reply:
also would it be possible to use use (just in this test as you refactor it either `testStorage.getNamespaceName()` or  `Environment.TEST_SUITE_NAMESPACE` as now it is mix.